ledger/simulation/tracer.go

```diff
@@ -267,10 +290,41 @@ func (tracer *evalTracer) BeforeOpcode(cx *logic.EvalContext) {
 			txnTrace = tracer.execTraceStack[len(tracer.execTraceStack)-1]
 		}
 		*txnTrace.programTraceRef = append(*txnTrace.programTraceRef, tracer.makeOpcodeTraceUnit(cx))
+
+		if tracer.result.ReturnStackChange() {
+			latestOpcodeTraceUnit := &(*txnTrace.programTraceRef)[len(*txnTrace.programTraceRef)-1]
+			latestOpcodeTraceUnit.appendDeletedStackValue(cx, tracer)
+			latestOpcodeTraceUnit.DisassembledLine, _ = cx.DisassembleLine()
```

Comment:
I see you added a disassembled line. Can you explain more about how this relates to reporting stack info?

Reply:
The disassembled line was just for debugging use, I wanted to make sure that the stack changes tie to right opcodes.

I have deleted them in [remove disassemble line](https://github.com/algorand/go-algorand/pull/5440/commits/caa07bbbef47d2b542dfbd0f84fb605881faab42).